apis/r/R/SOMASparseNDArray.R

```diff
@@ -222,7 +222,7 @@ SOMASparseNDArray <- R6::R6Class(
           identical(names(values), c(dnames, attrn))
       )
 
-      # Arrow Tables cannot have NULL names, so this only applies to data frames
+      # Arrow Tables cannot have NULL names, so this only applies to dataframes
```

Comment:
Actually you have 'data frame' throughout this PR, and if that's what people say in R land, that's fine ... I thought it was dataframe all one word & if I'm right you should do a search-and-replace on this PR ...

Reply:
Base-R uses "data frame" as seen in [`?data.frame`](https://stat.ethz.ch/R-manual/R-devel/library/base/html/data.frame.html)